echo.go

```diff
@@ -729,6 +749,11 @@ func handlerName(h HandlerFunc) string {
 	return t.String()
 }
 
+// PathUnescape is wraps `url.PathUnescape`
+func PathUnescape(s string) (string, error) {
```

Comment:
We can drop this function and directly use `url.PathUnescape` - wherever it is being used.